<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbFallasEditar</class>
 <widget class="QDialog" name="dbFallasEditar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Guardar">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Editar">
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Borrar">
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="FallaDescripcion">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formato de la version. Para numeros ingresar tantos 0 como digitos se necesiten, se puede intercalar con letras y simbolos. &lt;/p&gt;&lt;p&gt;Para formatos de version con letras &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>13</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Falla</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripcion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="FallaEdit">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>31</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formato de la version. Para numeros ingresar tantos 0 como digitos se necesiten, se puede intercalar con letras y simbolos. &lt;/p&gt;&lt;p&gt;Para formatos de version con letras &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="FallaBonif">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formato de la version. Para numeros ingresar tantos 0 como digitos se necesiten, se puede intercalar con letras y simbolos. &lt;/p&gt;&lt;p&gt;Para formatos de version con letras &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bonificacion</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FallaPresupuesto">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formato de la version. Para numeros ingresar tantos 0 como digitos se necesiten, se puede intercalar con letras y simbolos. &lt;/p&gt;&lt;p&gt;Para formatos de version con letras &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Falla Rep</string>
   </property>
  </widget>
  <widget class="QTableView" name="FallaTabla">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>601</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
  </widget>
  <widget class="QTableView" name="ProdTablaF">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="FallaBuscar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDate|Qt::ImhSensitiveData</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>80</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FallaEdit</tabstop>
  <tabstop>FallaDescripcion</tabstop>
  <tabstop>FallaPresupuesto</tabstop>
  <tabstop>FallaBonif</tabstop>
  <tabstop>Guardar</tabstop>
  <tabstop>Editar</tabstop>
  <tabstop>Borrar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dbFallasEditar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dbFallasEditar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
